public/data/config.json

```diff
@@ -27,7 +28,8 @@
     },
     "area": {
       "category": "Local authority",
```

Comment:
Looking at the UI, I wonder if we should change this and National Overview to `Region` to make it more obvious what searching will do.

Reply:
Happy to try it on staging and gather feedback